shared/chat/conversation/bottom-banner/container.tsx

```diff
@@ -16,19 +17,22 @@ type OwnProps = {
 type Props = {
   type: 'invite' | 'none' | 'broken'
   users: Array<string>
+  dismissed: boolean
   openShareSheet: () => void
   openSMS: (email: string) => void
+  onDismiss: () => void
   usernameToContactName: {[username: string]: string}
 }
 
 class BannerContainer extends React.PureComponent<Props> {
   render() {
     switch (this.props.type) {
       case 'invite':
-        return (
+        return this.props.dismissed ? null : (
```

Comment:
You could write to utilize short circuiting:

```suggestion
        return !this.props.dismissed && (
```

Different people have different opinions on the readability of this, but just wanted to point it out in case you didn't know. I think it's fine either way.

Reply:
i did it this way to avoid the double negative, but meh. sure. your way seems more idiomatic.